tesseract_command_language/include/tesseract_command_language/serialization.h

```diff
@@ -0,0 +1,54 @@
+#ifndef TESSERACT_COMMAND_LANGUAGE_SERIALIZATION_H
```

Comment:
License. Also maybe a description of what this file is.

Reply:
Sure, and thank you for mentioning pagmo it is a great reference to for this and this file may not be needed but still researching how they avoided this.

Reply:
added